cobaya/samplers/polychord/polychord.py

```diff
@@ -180,7 +180,7 @@ def initialize(self):
         locs = bounds[:, 0]
         scales = bounds[:, 1] - bounds[:, 0]
```

Comment:
There might remain some possible cleaning up to do, e.g. I expect that `self.logvolume` won't be needed anymore, right? Consequently also `scales` and `locs` can be removed. Maybe we don't even need `bounds` here anymore? Meaning we could even get rid of all lines from 170 to 181. 

Or is there a different reason that PolyChord priors need to be bounded that isn't obvious to me? @williamjameshandley, maybe you can confirm whether by using the `.ppf` we indeed don't need to specify prior bounds anymore?

Reply:
Certainly most other interfaces to PolyChord don't put in these bounds, so if it is necessary it might be cobaya specific. These changes all look good to me, so it's up to @JesusTorrado to approve/comment on this.